pkg/state/helmx.go

```diff
@@ -116,23 +102,15 @@ func (st *HelmState) PrepareChartify(helm helmexec.Interface, release *ReleaseSp
 	}
 
 	for _, d := range release.Dependencies {
-		chart := d.Chart
-		if st.fs.DirectoryExistsAt(chart) {
-			var err error
-
-			// Otherwise helm-dependency-up on the temporary chart generated by chartify ends up errors like:
-			//   Error: directory /tmp/chartify945964195/myapp-57fb4495cf/test/integration/charts/httpbin not found]
-			// which is due to that the temporary chart is generated outside of the current working directory/basePath,
-			// and therefore the relative path in `chart` points to somewhere inexistent.
-			chart, err = filepath.Abs(filepath.Join(st.basePath, chart))
-			if err != nil {
-				return nil, clean, err
-			}
+		dependenceChart, err := st.goGetterChart(d.Chart, "", release.CacheDir(), release.ForceGoGetter)
+
+		if err != nil {
+			return nil, clean, err
 		}
 
 		c.Opts.AdhocChartDependencies = append(c.Opts.AdhocChartDependencies, chartify.ChartDependency{
 			Alias:   d.Alias,
-			Chart:   chart,
+			Chart:   dependenceChart,
```

Comment:
Ohhh! How does this actually work?!

I remember that this Chart field is eventually transformed into the "chart" field in the temporary chart's dependencies section.

Does Helm have support for letting charts at absolute paths be the dependencies of a chart?

Reply:
Work in progress. What you said is the key issue. 😃 I need to confirm that.

Reply:
Maybe we can use file:///?

Reply:
I can see some possibility of success for the use of file URLs!

Reply:
Or perhaps we could try to transform cache paths generated by the goGetterChart func to relatives ones so that helm can process them?

Reply:
@mumoshu I will try to do as per your comments.